scripts/missions/cop/4_3_The_Secrets_of_Worship.lua

```diff
@@ -119,28 +140,17 @@ mission.sections =
                 end,
             },
 
-            ['qm_professor_mariselle'] =
-            {
-                onTrigger = function(player, npc)
-                    local missionStatus = mission:getVar(player, 'Status')
-                    local isSpawnPoint = npc:getLocalVar('hasProfessorMariselle') == 1
-
-                    if
-                        missionStatus == 3 and
-                        not player:hasKeyItem(xi.ki.RELIQUIARIUM_KEY) and
-                        isSpawnPoint and
-                        npcUtil.popFromQM(player, npc, sacrariumID.mob.OLD_PROFESSOR_MARISELLE, { radius = 2, hide = 0 })
-                    then
-                        return mission:messageSpecial(sacrariumID.text.EVIL_PRESENCE)
-                    elseif
-                        mission:getLocalVar(player, 'hasKilled') == 1 and
-                        not player:hasKeyItem(xi.ki.RELIQUIARIUM_KEY)
-                    then
-                        npcUtil.giveKeyItem(player, xi.ki.RELIQUIARIUM_KEY)
-                        return mission:noAction()
-                    end
-                end,
-            },
+            ['qm_prof_0'] = profQmOnTrigger,
+
```

Comment:
Just a small styling thing, I would remove the newlines between the qm_prof NPCs just to tidy things up, since they're only one-liners.

Reply:
For this I'm referring to changing it to the below:
```
            ['qm_prof_0'] = profQmOnTrigger,
            ['qm_prof_1'] = profQmOnTrigger,
            ['qm_prof_2'] = profQmOnTrigger,
            ['qm_prof_3'] = profQmOnTrigger,
            ['qm_prof_4'] = profQmOnTrigger,
            ['qm_prof_5'] = profQmOnTrigger,
```